readme.md

```diff
@@ -395,6 +395,7 @@
 - [CouchDB](https://github.com/quangv/awesome-couchdb#readme) - Document-oriented NoSQL database.
 - [HBase](https://github.com/rayokota/awesome-hbase#readme) - Distributed, scalable, big data store.
 - [NoSQL Guides](https://github.com/erictleung/awesome-nosql-guides#readme) - Help on using non-relational, distributed, open-source, and horizontally scalable databases.
+- [Database tools](https://github.com/mgramin/awesome-db-tools) - List of database tools (IDE, CLI, managing, monitoring, migrations, modelers, visualization etc).
```

Comment:
The title should be titleized.

The word `List` is implied. All Awesome lists are lists...

`database tools` is just repeating the the tile.

Please write a better description.